### PR TITLE
fix trip logs issue

### PR DESCRIPTION
**Pull request by crystalzha0**
EDIT:
1. Remove GetTrip log fetching which improves the fleet debugger UI performance significantly
2. Minor cleanup

This PR doesn't actually fix any other trip logs issue.

### CODE REVIEW

File: components/fleetArchiveDS.js [hunk 1]
```diff
@@ -141,7 +141,6 @@ class FleetArchiveLogs extends Datasource {
       .map(
         (entry) =>
           _.get(entry, "createTrip.response.vehicleId") ||
-          _.get(entry, "getTrip.response.vehicleId") ||
```
Comment (ewbao) [inline components/fleetArchiveDS.js:144]: I don't think Cloud Logging filters out getTrip calls? Can you tell me more?

File: components/fleetArchiveDS.js [hunk 2]
```diff
@@ -147,9 +147,6 @@ class FleetArchiveLogs extends Datasource {
       .uniq()
       .compact()
       .value();
-    if (vehicleIds.length === 0) {
```
Comment (ewbao) [inline components/fleetArchiveDS.js:150]: What does this fix? It's a bit unclear.

File: components/fleetArchiveDS.js (hunk 1 above)
Comment (ewbao) [reply components/fleetArchiveDS.js:144]: Nevermind, I see a filter here https://github.com/googlemaps/fleet-debugger/blob/5f67d794f24aa276ad97ee72368a987861f65731/components/logging.js#L79

File: components/fleetArchiveDS.js (hunk 2 above)
Comment (crystalzha0) [reply components/fleetArchiveDS.js:150]: You are right. This doesn't actually fix anything. It just removes an early exit.

File: components/fleetArchiveDS.js (hunk 1 above)
Comment (crystalzha0) [reply components/fleetArchiveDS.js:144]: Yes, also get_trip isn't in the "interestingLogs" list so the fetching code already might not be getting them anymore.

**Pull request closed without merging**